playbooks/roles/topbeat/templates/topbeat.yml.j2

```diff
@@ -0,0 +1,19 @@
+---
+input:
+  # In seconds, defines how often to read server statistics
+  period: {{ topbeat_period_seconds }}
+
+  # Regular expression to match the processes that are monitored
+  # By default, all the processes are monitored
+  procs: {{ TOPBEAT_PROCS }}
+
+  # Statistics to collect (all enabled by default)
```

Comment:
Maybe it's me, but I keep getting confused by this wording.

When I see some form of "the default configuration is BLAH" followed by
some explicit configuration, I assume that that we must be diverging from
the defaults, else why would we bother redefining the defaults?

This also presents a potential maintenance burden. If topbeat adds a new
stat collection and we don't immediately update this field, a reader
would mistakenly think `[system, proc, filesystem]` was the
comprehensive list, whereas it no longer was.

In the current context, we shouldn't mention defaults if we're
explicitly overriding them (like here), because the defaults no longer
matter. Even if defaults change, we're "pinning" the list here.

Do we even need to define this? If we removed this entire stanza, would
topbeat load the proper defaults anyway?

That said, we may want to selectively disable stats in certain
environments, so we're probably better served by parameterizing this:

```yaml
# templates/topbeat.yml.j2
stats: {{ topbeat_stats}}
```

```yaml
# defaults/main.yml
TOPBEAT_STATS_FILESYSTEM: true
TOPBEAT_STATS_PROC: true
TOPBEAT_STATS_SYSTEM: true
topbeat_stats:
  filesystem: {{ TOPBEAT_STATS_FILESYSTEM }}
  proc: {{ TOPBEAT_STATS_PROC }}
  system: {{ TOPBEAT_STATS_SYSTEM }}
```

The reason it's important to compose the dictionary like this (creating
a top-level variable that is itself composed of other top-level
variables) is so that we can easily override single entries at deploy-time.

It allows us to do something like:
```sh
# This would turn off process monitoring for lms/cms
ansible-playbook edxapp.yml -e 'TOPBEAT_STATS_PROC=false'
```

And just to point it out, this is where the ALL_CAPS variables versus
all_lower makes intent more obvious:
- `TOPBEAT_STATS_FILESYSTEM`, `TOPBEAT_STATS_PROC`, and
  `TOPBEAT_STATS_SYSTEM` are all intended to be overridden.
- `topbeat_stats`, the composite value, should be generally left as-is.